data/human/deep jobs.txt

```diff
@@ -403,27 +409,27 @@ mission "Secure Transport Job (Secret) [0]"
 	to offer
 		random < 25
 		not "deep security reveal"
-		"reputation: Deep Security" >= -50
+		"reputation: Deep Security" >= 0
+		"reputation: Deep Security" >= - "secure deep package" * 3
```

Comment:
Isn't this line redundant with the one above it? If your reputation is above 0 then it's never negative, so the second line doesn't matter. If your reputation is negative, then you'll be blocked by the first line.

Reply:
The first line is a sanity check to make sure that Deep Sec aren't offering jobs to someone they hate. The second one is to make players who've failed a secure transport job need to raise their rep before they can be trusted another one again (and that line would allow negative rep players to still be offered secure transports if "secure deep package" is positive).

Reply:
Sounds like you're missing an `or`.

Reply:
My thought process behind having both conditions:
* When "secure deep package" is 0: `- "secure deep package" * 3` is also 0, so the player is offered these missions when above negative rep.
* When "secure deep package" is positive: `- "secure deep package" * 3` is negative, but the first line stops players with negative rep from being offered the job anyway.
* When "secure deep package" is negative (through failing jobs): `- "secure deep package" * 3` is positive, meaning that players who've failed a secure transport job need to increase their rep with Deep Security before they can be offered the missions again.

Reply:
That checks out, then.